flogging/flogging.py

```diff
@@ -235,12 +236,17 @@ def setup(
     :param config_path: Path to a yaml file that configures the level of output of the loggers. \
                         Root logger level is set through the level argument and will override any \
                         root configuration found in the conf file.
+    :param trailing_dot: If False, raise an exception when a logging message ends with a dot.
     :return: None
     """
     global logs_are_structured
     logs_are_structured = structured
 
     level = level if level is not None else os.environ.get("LOG_LEVEL", "INFO")
+    structured = structured if structured is not None else os.environ.get("LOG_STRUCTURED", False)
+    trailing_dot = (
+        trailing_dot if trailing_dot is not None else os.environ.get("LOG_TRAILING_DOT", False)
+    )
```

Comment:
This will never be configured like this. It is always either yes or true, nobody is going to support both.

Reply:
Yeah you are right. I removed that already.